cli/src/logger/mod.rs

```diff
@@ -1,12 +1,38 @@
+#[repr(u8)]
+pub enum ErrorKind {
```

Comment:
Nit: just `Error` or `ErrorType` is what I usually see?

Reply:
Or maybe ErrorCode, more similar to `deploy`?

Reply:
Will go with `ErrorType`